src/services/apis/parseGanaCode.js

```diff
@@ -0,0 +1,11 @@
+export const parseKatakanaToHiragana = oriStr => {
+  return oriStr
+    .split("")
+    .map(letter => {
+      const code10 = letter.charCodeAt(0);
+      return code10 > 12448 && code10 < 12543 // Unicode range for Katakana
```

Comment:
Personally, I'd suggest writing `12448` and `12543` as constants to increase readability.
For example,
```js
const KATAKANA_START = 12448;
const KATAKANA_END = 12543;
// ...
const codeUnit = letter.charCodeAt(0);
return codeUnit > KATAKANA_START && codeUnit < KATAKANA_END
```

So that you don't have to write comments because the code explains itself.

Reply:
Very cool, will do